logging/log.go

```diff
@@ -45,14 +48,15 @@ func (f *prefixFormatter) Format(e *logrus.Entry) ([]byte, error) {
 	return append([]byte(f.prefix), b...), nil
 }
 
-func initApplicationLog(prefix string, output io.Writer) {
-	if prefix != "" {
-		logrus.SetFormatter(&prefixFormatter{
-			prefix, logrus.StandardLogger().Formatter})
+func initApplicationLog(o Options) {
```

Comment:
Here is an example output of `bin/skipper -application-log-json-enabled`:
```json
{"level":"info","msg":"Expose metrics in codahale format","time":"2020-08-05T11:54:50+02:00"}
{"level":"warning","msg":"no route source specified","time":"2020-08-05T11:54:50+02:00"}
{"level":"info","msg":"support listener on :9911","time":"2020-08-05T11:54:50+02:00"}
{"level":"info","msg":"proxy listener on :9090","time":"2020-08-05T11:54:50+02:00"}
{"level":"info","msg":"TLS settings not found, defaulting to HTTP","time":"2020-08-05T11:54:50+02:00"}
```

There are a couple of things to clarify/polish:
* Is it ok to ignore `ApplicationLogPrefix` when JSON format is enabled?
* Is it ok that log level is lowercase?
* Access log uses custom datetime format presumably to match apache's log format. Is it ok to output timestamps in default logrus' format (`time.RFC3339`)?
* Access log uses customs time field name (`timestamp`). Is it ok to use default logrus' name (`time`)?